StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -289,17 +289,52 @@ public class STPPaymentHandler: NSObject {
         with authenticationContext: STPAuthenticationContext,
         returnURL: String?,
         completion: @escaping STPPaymentHandlerActionPaymentIntentCompletionBlock
+    ) {
+        if !STPPaymentIntentParams.isClientSecretValid(paymentIntentClientSecret) {
+            completion(.failed, nil, _error(for: .invalidClientSecret))
+            return
+        }
+        apiClient.retrievePaymentIntent(
+            withClientSecret: paymentIntentClientSecret,
+            expand: ["payment_method"]
+        ) { [weak self] paymentIntent, error in
+            guard let self else {
+                return
+            }
+            if let paymentIntent = paymentIntent, error == nil {
+                self.handleNextAction(for: paymentIntent, with: authenticationContext, returnURL: returnURL, completion: completion)
+            } else {
+                completion(.failed, paymentIntent, error as NSError?)
+            }
+        }
+    }
+
+    /// (Internal) Handles any `nextAction` required to authenticate the PaymentIntent.
+    /// Call this method if you are using server-side confirmation.  - seealso: https://stripe.com/docs/payments/accept-a-payment?platform=ios&ui=custom
```

Comment:
I copy and pasted this from the public API, I'm not sure why it links to CardElement or what we could link to instead... I don't think it should link to the "Finalize payments on a server" since merchants don't ever call this method in that integration though.